pagopa/io-app
# feat: [IOCOM-1417] Security suggestions bottom sheet, new messages' home

⚠️ This PR depends on #5894 ⚠️

## Short description
This PR adds the security suggestions bottom sheet to the new messages' home.

|Security Suggestions<br/>iOS|Security Suggestions<br/>Android|
|-|-|
|<video src="https://github.com/pagopa/io-app/assets/5150343/255d8fe5-c1e1-4ff3-ba76-ae03a26e7ad8">|<video src="https://github.com/pagopa/io-app/assets/5150343/1fc3c932-cc7d-43a0-bc17-87dcdd124fa7">|

## List of changes proposed in this pull request
- `useSecuritySuggestionBottomSheet` wrapped inside a `SecuritySuggestions` component (linked in `MessagesHomeScreen`) in order to avoid multiple messages section re-renderings

## How to test
Using the io-dev-api-server, make sure to enable fast-login. Since the new messages' home requires two flags to be active and the security suggestions bottom sheet does not appear a second time, fastest way to test is to add the following code to `LandingScreen.tsx`:

```
  const { setExperimental } = useIOExperimentalDesign();
  ..
  React.useEffect(
   () => () => {
      ..
      AsyncStorage.setItem(DS_PERSISTENCE_KEY, JSON.stringify(true)).finally(
        () => {
          dispatch(
            preferencesDesignSystemSetEnabled({ isDesignSystemEnabled: true })
          );
          setExperimental(true);
          dispatch(
            preferencesNewHomeSectionSetEnabled({
              isNewHomeSectionEnabled: true
            })
          );
        }
      );
    },
    []
  );
```
